win32ss/gdi/eng/mdevobj.c

```diff
@@ -0,0 +1,202 @@
+/*
+ * COPYRIGHT:        See COPYING in the top level directory
+ * PROJECT:          ReactOS kernel
+ * PURPOSE:          Support for meta devices
+ * FILE:             win32ss/gdi/eng/mdevobj.c
+ * PROGRAMER:        Hervé Poussineau
+ */
+
+#include <win32k.h>
+#define NDEBUG
+#include <debug.h>
+DBG_DEFAULT_CHANNEL(EngMDev);
+
+PMDEVOBJ gpmdev = NULL; /* FIXME: should be stored in gpDispInfo->pmdev */
+
+VOID
+MDEVOBJ_vEnable(
+    _Inout_ PMDEVOBJ pmdev)
+{
+    ULONG i;
+
+    for (i = 0; i < pmdev->cDev; i++)
```

Comment:
codestyle for for-loops is to always put their body inside braces.

Reply:
Done